components/pages/explorer/PageContent.tsx

```diff
@@ -67,6 +79,15 @@ const PageContent = () => {
 		firstRender || isEqual(sqon, currentFilters) || setCurrentFilters(sqon);
 	}, [currentFilters, firstRender, setCurrentFilters, sqon]);
 
+	const fileTableActive = tableType === tableTypes['REPO_TABLE'];
```

Comment:
there's a standard convention to do `isWhatever` for boolean values.
without "is", then the expected value could be an enum, leading to "this variable says WHICH fileTable is active"
```suggestion
	const isFileTableActive = tableType === tableTypes['REPO_TABLE'];
```

Reply:
With multiple visualizers (JBrowse for example) this value will likely no longer be a boolean
I originally had the original var name as `isActive` but that was too vague, `isFileTableActive` communicates everything